### PR TITLE
MichaelMSinclair adds hometown of Toronto.

### PR DESCRIPTION
**Pull request by michaelmsinclair**
Oddly enough there are amusement parks around the city. Toronto isn't as dull as many people think.

Let's add Toronto to the the list of hometowns. 

Questions:

1. If you've never visited Toronto, will you visit soon?
2. If you *have* been to Toronto, what was the most memorable thing you saw or did there?
3. What's your favourite food? I'm sure we can find a restaurant that serves it.

### CODE REVIEW

File: michaelmsinclair-toronto.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Toronto, Ontario, Canada
+
+### Restaurants
```
Comment (dbiscoIntuit) [inline michaelmsinclair-toronto.md]: @michaelmsinclair I ❤️ Chinese food. Do you have any Chinese restaurant recommendations?

Comment (michaelmsinclair) [inline michaelmsinclair-toronto.md]: There are tonnes of Chinese Restaurants in and around Toronto.

Comment (DanSewell) [inline michaelmsinclair-toronto.md]: I have been to Toranto, but I didn't find any good Ice-Cream places.  Is Ice-Cream big up there?

Comment (michaelmsinclair) [inline michaelmsinclair-toronto.md]: > Toranto,

You need to invest in autocorrect awareness.

**Comment by michaelmsinclair** [comment]
I've added Chinese restaurants, and will add Ice Cream stores once I've "done more research".

**Comment by dbiscoIntuit** [comment]
@michaelmsinclair 👍  go ahead and 🚢

**Pull request merged**